### PR TITLE
Shushman/1.0 fixes

### PR DESCRIPTION
**Pull request by Shushman**
Tests are having trouble due to problems in registered version of `Compose`?

### CODE REVIEW

File: src/Reel.jl [hunk 1]
```diff
@@ -54,13 +52,13 @@ const mime_ordering = map(MIME, [
 
 function bestmime(x)
     for m in mime_ordering
-        if mimewritable(m, x)
+        if show:able(m, x)
```
Comment (tawheeler) [inline src/Reel.jl]: mistype?

**Comment by Shushman** [comment]
I get the following error from the last cell of the `PGFPlots.jl` [notebook](https://github.com/JuliaTeX/PGFPlots.jl/blob/master/doc/PGFPlots.ipynb)

```
objects of type getfield(Reel, Symbol("##2#4")) cannot be finalized

Stacktrace:
 [1] error(::String, ::Type, ::String) at ./error.jl:42
 [2] finalizer(::Any, ::Any) at ./gcutils.jl:16
 [3] (::getfield(Reel, Symbol("##call#1#3")))(::Int64, ::Type{Frames{MIME{Symbol("image/svg+xml")}}}) at /Users/shushmanchoudhury/.julia/dev/Reel/src/Reel.jl:20
 [4] (::getfield(Core, Symbol("#kw#Type")))(::NamedTuple{(:fps,),Tuple{Int64}}, ::Type{Frames{MIME{Symbol("image/svg+xml")}}}) at ./none:0
 [5] #Frames#5(::Int64, ::Type, ::MIME{Symbol("image/svg+xml")}) at /Users/shushmanchoudhury/.julia/dev/Reel/src/Reel.jl:24
 [6] (::getfield(Core, Symbol("#kw#Type")))(::NamedTuple{(:fps,),Tuple{Int64}}, ::Type{Frames}, ::MIME{Symbol("image/svg+xml")}) at ./none:0
 [7] top-level scope at none:0
```

**Comment by MaximeBouton** [comment]
The tests depend on Gadfly and Compose, let's wait for these PR to be merged: 
- Gadfly GiovineItalia/Gadfly.jl#1189
- Compose GiovineItalia/Compose.jl#282

**Comment by MaximeBouton** [comment]
The test using Compose is actually commented out, should we just remove it and remove the dependency? 
These two PR on Gadfly and Compose seem to take longer than expected, should we just write tests with another plot backend?

**Comment by Shushman** [comment]
Yeah I recall that there was an issue with the `Compose` test even in `Julia 0.6`. I can put together a fix with a different plotting backend.

**Comment by Shushman** [comment]
@tawheeler bump!

File: REQUIRE [hunk 2]
```diff
@@ -1,2 +1,2 @@
-julia 0.6
-VideoIO 0.1.0
+julia 0.7
```
Comment (tawheeler) [inline REQUIRE]: Just to confirm: this is 0.7 because it is the _minimum_ requirement, this it works on 1.0 as well.

Comment (Shushman) [reply REQUIRE]: yeah, that is the idea, right?

Comment (tawheeler) [reply REQUIRE]: yup you're good

Comment (Shushman) [reply REQUIRE]: Great! You still need to merge though, I don't have write access to the upstream :P

**Comment by tawheeler** [comment]
Okay, I actually don't have admin rights to add you, but I can merge it.

**Pull request merged**